otel-worker-core/src/data/sql.rs

```diff
@@ -35,8 +35,11 @@ impl SqlBuilder {
     ///
     /// Query parameters:
     /// - $1: trace_id
-    pub fn span_list_by_trace(&self) -> String {
-        String::from("SELECT * FROM spans WHERE trace_id=$1")
+    pub fn span_list_by_trace(&self, limit: Option<u32>) -> String {
```

Comment:
I would prefer if this limit is set to None, that it would actually not add a limit at all. We should move the default limit = 100 to where ever it is called from.

Reply:
Actually maybe it is better to keep it here, since we're not doing any streaming